server/game/game.js

```diff
@@ -766,9 +766,17 @@ class Game extends EventEmitter {
         }
     }
 
+    checkForTimeExpired() {
+        if(this.timeLimit.isTimeLimitReached && !this.finishedAt) {
+            this.addAlert('success', 'The game has ended because the timer has expired.  Timed wins are not currently implemented');
```

Comment:
It doesn't seem like it would be very complicated to record / announce the winner. The most complicated case is if Tycho Nestoris is in play.

Though we could certainly do that as a separate PR. Unless there's concern tying timed win rules to a particular tournament format?

Reply:
I think EvadLhorg is working on that so I didn’t want to step on his toes :)